package.json

```diff
@@ -80,5 +80,8 @@
   },
   "resolutions": {
     "bcrypto": "5.2.0"
+  },
+  "dependencies": {
```

Comment:
Hey are we using `stylis`? can be removed if not.

Reply:
It's referenced by `emotion` and without this I get the error [`stylis.middleware is not a function`](https://github.com/emotion-js/emotion/issues/2138).

Reply:
I see. Then would it make sense to add it under `design-system` where `emotion` is declared ?

Reply:
I think I tried that and it didn't resolve the issue, but I'll try it that way again just to be sure.

Reply:
I removed it and, for the time being, I'm not getting the middleware error, so I just left it out.